Examples/Example-tvOS/Example-tvOS/AppAuthTVExampleViewController.m

```diff
@@ -29,18 +33,6 @@
  */
 static NSString *const kClientSecret = @"YOUR_CLIENT_SECRET";
 
-/*! @brief Device authorization endpoint.
- */
-static NSString *const kTVAuthorizationEndpoint = @"https://oauth.example.com/device";
-
-/*! @brief Token endpoint.
- */
-static NSString *const kTokenEndpoint = @"https://oauth.example.com/token";
-
-/*! @brief User info endpoint.
- */
-static NSString *const kUserInfoEndpoint = @"https://oauth.example.com/userinfo";
-
 /*! @brief NSCoding key for the authorization property.
  */
 static NSString *const kExampleAuthorizerKey = @"authorization";
```

Comment:
I wonder if the sample should offer both configuration options like some of the others. The problem with relying on discovery is that not all providers offer it. It wouldn't necessarily need to be a separate button, be a choice the developer makes in the code.

Reply:
+1